third-party/snyk/snapshot.json

```diff
@@ -101,7 +99,7 @@
       {"id":"npm:knockout:20130701","severity":"medium","semver":{"vulnerable":[">=2.1.0-pre <3.0.0"]}}
     ],
     "lodash":[
-      {"id":"SNYK-JS-LODASH-567746","severity":"medium","semver":{"vulnerable":["<=4.17.15"]}},
```

Comment:
this was a deliberate change discussed in the roll: https://github.com/GoogleChrome/lighthouse/pull/11144/files#r459028283